### DISCUSSION TITLE
Some feedback regarding faces and stacking

### DISCUSSION THREAD

**Discussion by loulou91** [photoprism/photoprism#3464] [Q&A]
After a first scanning of a first part of my originals (about 25k) I tagged all my faces (long and tedious process) but at the end of the day some dozen of faces were still considered as new faces in people screen. Some were without any face some with correctly tagged faces.
After large investigation, I discovered that these pictures were stacked. To be noted that stacking feature is activated by default.

Afterwards I disabled stack feature, I suppressed all stacked pictures and launched rescan after reimporting deleted pictures. Then face tagging was successfully completed.

For the time being I still have not clearly understood stacking feature and the interest... So, it's now deactivated and it works fine for me and I see all my pictures correctly tagged.

**Comment by artincube** [comment]
oh ok. I just posted somewhere else about the same. we got both there.

Showing face:new and not face:yes in search while "apparently" in the people tab all is already named is not expected.

**Comment by graciousgrey** [comment]
That's a known issue: https://docs.photoprism.app/known-issues/#photos-with-all-faces-assigned-appear-under-new-faces 

To solve it you can
(A)  unstack the files, if you don't like the stacking
(B) Change the primary file and add names to the faces of this file

> For the time being I still have not clearly understood stacking feature and the interest

Photos can be related to each other even if they are not exact duplicates
- e.g. different edits of a photo (color, sepia,...)
- different qualities of a photo
- different formats
- time series photos

Many of our users don't want their photo views to be "polluted" by many variants of more or less the same photo.
